cylc/flow/remote.py

```diff
@@ -154,6 +155,66 @@ def construct_platform_ssh_cmd(raw_cmd, platform, **kwargs):
     return ret
 
 
+def get_includes_to_rsync(rsync_includes=None):
+    """Returns a list of directories/files, configured in suite.rc,
```

Comment:
s/suite.rc/flow.cylc ? I **think**?

Reply:
:) thanks @kinow, I suspect that habit will take a while to break!

Reply:
Thanks for spotting @kinow, I'll get that changed - hard habit to break! Btw, was just on my son's computer (for the vc) and just replied to this but was his github account lol. Sorry if you got an email notification from a from a strange account - 'twas me being a lemon!

Reply:
I got a notification, but when I checked here there was nothing. Interesting, I think that's what happens when you remove a comment. The other person might sill have a notification, but just won't see anything. :nerd_face:

Reply:
Yep, comments for this pr now popping up on his computer hehe.